packages/material/scss/calendar/_layout.scss

```diff
@@ -52,7 +52,7 @@ $calendar-view-gap: 48px !default;
                 6 * $calendar-month-cell-size,
                 $calendar-weekdays-height,
                 -3 * $calendar-meta-cell-size,
-                .5
+                .45
```

Comment:
This should be reverted. The previous logic is precise and correct.

Reply:
I do believe it is neither precise nor correct. 

As you can see in the below video:
https://www.screencast.com/t/Cy8w7P30CwWI

The height of the "k-month" table is 276px. On the other hand, the height of the "k-meta-view" is 240px. This makes me believe that we should add 36px in padding (2*18px), however, we add 2*20px.